### PR TITLE
Feature/introduce axa footer small

### PR DESCRIPTION
**Pull request by raphaellueckl**
Changes proposed in this pull request:

 - Implement the small axa.ch header as webcomponent

## Type of change

- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by LucaMele** [comment]
Im not commenting

**Comment by raphaellueckl** [comment]
> Im not commenting

*Report abuse...*

**Comment by raphaellueckl** [comment]
Please have a look. This is only a "draft" so far, but one that fully works. Please be picky.

File: src/components/molecules/footer-small/index.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    &--left {
+      padding-right: 5px;
+    }
+
+    &--right {
+      padding-left: 5px;
+    }
+  }
+
+  p {
```
Comment (LucaMele) [inline src/components/molecules/footer-small/index.scss]: please not use globals

File: src/components/molecules/footer-small/index.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    font-family: 'Source Sans Pro', Arial, sans-serif;
+    font-size: 13px;
+    font-weight: 400;
+  }
+
+  a {
```
Comment (LucaMele) [inline src/components/molecules/footer-small/index.scss]: please not use globals

**Comment by raphaellueckl** [comment]
Please have a look again

**Comment by jackblackCH** [comment]
The team decided to start commit messages with capital letter.

**Comment by jackblackCH** [comment]
There was probably something wrong with the rebase, lot's of commits from other braches are in here and visible as file change.

File: src/components/20-molecules/footer-small/index.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.footer-small {
```
Comment (jackblackCH) [inline src/components/20-molecules/footer-small/index.scss]: We prefix it with the component type for molecules `m-` -> `.m-footer-smal`

**Pull request closed without merging**